feature-libs/user/profile/occ/adapters/occ-user-profile.adapter.ts

```diff
@@ -39,7 +41,9 @@ export class OccUserProfileAdapter implements UserProfileAdapter {
   constructor(
     protected http: HttpClient,
     protected occEndpoints: OccEndpointsService,
-    protected converter: ConverterService
+    protected converter: ConverterService,
+    protected captchaConfig?: CaptchaApiConfig,
```

Comment:
Lets use `inject` to include these services instead like the `LoggerService` above. This makes it easier for us to use our schematics when upgrading (ie. its easier to bump injectors than modify constructor).

```
protected captchaConfig = inject(CaptchaApiConfig, { optional: true});
protected injector = inject(Injector, { optional: true});
```
Note: Optional flag would be removed when we bump major versions.

Reply:
Thanks, Changed.